src/doc/01 - intro.rst

```diff
@@ -5,6 +5,10 @@ Introduction
 Problem Statement: vendor lock-in
 ---------------------------------
 
+I want to add something there.
+
+This is better with this sentence.

```

Comment:
I'm not sure this is better. Can you double check?

Reply:
Sorry, I wil try again

Reply:
Thanks

Reply:
nice!